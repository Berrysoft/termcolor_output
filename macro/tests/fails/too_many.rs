use termcolor::NoColor;
use termcolor_output::colored;

fn main() {
    let mut w: NoColor<Vec<u8>> = NoColor::new(vec![]);
    match colored!(w, "", Some(()), vec![1u8]) {
        Ok(_) => {}
        Err(_) => {}
    };
}
