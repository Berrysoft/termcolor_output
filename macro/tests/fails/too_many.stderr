error: Too many input parameters for this format string
 --> $DIR/too_many.rs:6:27
  |
6 |     match colored!(w, "", Some(()), vec![1u8]) {
  |                           ^^^^^^^^
